### PR TITLE
Add instructions for using Bitbucket in pipeline

### PR DESCRIPTION
**Pull request by gitlawr**
Address issue: https://github.com/rancher/docs/issues/1009

### CODE REVIEW

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
Please add `_Available as of v2.2.0_` so that people know when these features were available.

https://github.com/rancher/docs/pull/1199/files#diff-8ad9c71b76b83c64e18dda58fa1e8c68R9

**Comment by loganhz** [comment]
LGTM

File: content/rancher/v2.x/en/tools/pipelines/configurations/_index.md [hunk 1]
```diff
@@ -75,6 +75,54 @@ Select your provider's tab below and follow the directions.
...
+    ![Bitbucket Server Pipeline Instructions]({{< baseurl >}}/img/rancher/bitbucketserver-pipeline.png)
+
+1. Enter the host address of your Bitbucket server installation.
+
+1. Click **Authenticate**.
+
+1. Enable the repository for which you want to run a pipeline. Then click **Done**.
+
+>**Note:**
+> Bitbucket server needs to do SSL verification when sending webhooks to Rancher, please ensure that Rancher server's certificate is trusted by the Bitbucket server. There are two options:
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: Change "webhooks to Rancher, please ensure " to "webhooks to Rancher. Please ensure "

Comment (gitlawr) [reply content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: Updated

**Comment by loganhz** [comment]
No need to mention the Resources > Pipeline stuff in this RP as Bitbucket support only available in 2.2.0

**Pull request merged**